src/txFactory/credentialRegistryTransactionFactory.ts

```diff
@@ -67,7 +67,7 @@ export function getSubjectCredentialStatus(
   didSubject,
   subjectCredentialHash
 ) {
-  const subjectAddr = AIdUtils.getProxyAddress(didSubject)
+  const subjectAddr = `0x${AIdUtils.getProxyAddress(didSubject)}`
```

Comment:
I think that `AIdUtils.etProxyAddress()` should return the address with the `0x` as every address in the blockchain starts with `0x`.

Reply:
Yeah, i changed that now. I was testing things to make all work and i forgot to do that :sweat_smile: